lib/PageInteractions.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: TBD
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>
+
+package PageInteractions;
+use strict;
+use warnings FATAL => 'all';
+
+use testapi;
+use parent 'Exporter';
+use Exporter;
+
+our @EXPORT = qw(
+    find
+    send_keys
+    fill_in_password
+    is_displayed
+);
+
+sub find {
+    my $self = @_;
+    assert_screen(@_);
+}
+
+sub send_keys {
+    my $key = shift;
+    my $message = shift || "Send $key key";
+    record_info('send_key', $message);
```

Comment:
I would write the shortcut instead of the name of the function. It is quite visually annoying to read many of all those boxes with "send_keys", but $message is really useful. Other option, just use "Press key" but I guess it would kind of overkill me visually. Perhaps we need some enhancement in os-autoinst with this boxes, instead of creating a box, in the point when the api is used to be able to add the description in $message.